### PR TITLE
WIP fix store() default value bug

### PR DESCRIPTION
**Pull request by lucifergit**
fix store() default value creation'index. html'file path error,like '20190810index.html'

Each pull request should address a single issue, and have a meaningful title.

**Description**
Explain what you have changed, and why.

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPdocs
- [x] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

---------Remove from here down in your description----------

**Notes**
- Pull requests must be in English
- If the PR solves an issue, reference it with a suitable verb and the issue number
(e.g. fixes <hash>12345
- Unsolicited PRs will be considered, but there is no guarantee of acceptance
- Pull requests should be from a feature branch in the contributor's fork of the repository
  to the develop branch of the project repository

### CODE REVIEW

**Comment by lonnieezell** [comment]
This caused a problem with existing tests, and looks like file naming elsewhere was thrown off because of it. See https://travis-ci.org/codeigniter4/CodeIgniter4/jobs/570127900

File: system/HTTP/Files/UploadedFile.php [hunk 1]
```diff
@@ -400,12 +400,12 @@ public function isValid(): bool
 	 */
 	public function store(string $folderName = null, string $fileName = null): string
 	{
-		$folderName = $folderName ?? date('Ymd');
+		$folderName = $folderName ?? date('Ymd') . DIRECTORY_SEPARATOR;
```
[the remark this answers is not in the document]
Comment (jim-parry) [reply system/HTTP/Files/UploadedFile.php]: I made this change. Travis-CI passes, but code coverage down. Not sure how trustworthy coveralls is, hmmm.

**Comment by jim-parry** [comment]
I checked the user guide, and getErrorMessage is actually not mentioned on the validation page. It is an internal method.
I think we are good to go with this PR.

**Comment by MGatner** [comment]
Looks good to me! Coveralls has been weird - I don't know enough about how it does the count, but it seems like these days any PR that touches a file without *adding* tests will count as a decrease.

**Pull request merged**